bellows/zigbee/application.py

```diff
@@ -256,6 +257,7 @@ def request(self, nwk, profile, cluster, src_ep, dst_ep, sequence, data, expect_
 
     def permit(self, time_s=60):
         assert 0 <= time_s <= 254
+        yield from self.send_zdo_broadcast(0x0036, 0x0000, 0x00, [time_s, 0])
```

Comment:
This method is now a little weird -- it's a generator which also returns a coroutine. After #114, I think this should be written something like

```
async def permit(self, ...):
    assert ...
    await self.send_zdo_broadcast(...)
    return await self._ezsp.permitJoining(...)
```

Reply:
right, will do this